ios/RNSScreen.mm

```diff
@@ -103,6 +128,50 @@ - (UIViewController *)reactViewController
 }
 #endif
 
+- (void)setKeyboardSize:(CGSize)size
+{
+  NSLog(@"Setting keyboard size to %@", NSStringFromCGSize(size));
+  _keyboardSize = size;
+  //  [self updateFooterLayout];
```

Comment:
This also should be removed, I guess? Also, is this system method? Maybe it could be possible to just do `_keyboardSize = size;`?